models/track_and_reid_model.py

```diff
@@ -112,16 +112,36 @@ def get_reid_score(track_im_conf, distmat, g_pids):
 
     return ids_score
 
+def get_reid_score_mult_ids(track_im_conf, simmat, g_pids):
+    print(simmat.shape)
+    ids_score = {pid: 0 for pid in ID_TO_NAME.keys()}
+    aligned_simmat = (track_im_conf[:, np.newaxis] * ((simmat + 1)/2)) ** 5
```

Comment:
let's move the degree p to a const variable at the beginning of the file so we don't need to change all the places every time